es-app/src/views/gamelist/GridGameListView.cpp

```diff
@@ -71,7 +88,25 @@ GridGameListView::GridGameListView(Window* window, FileData* root) :
 	mDescription.setFont(Font::get(FONT_SIZE_SMALL));
 	mDescription.setSize(mDescContainer.getSize().x(), 0);
 	mDescContainer.addChild(&mDescription);
-
+	
+	mMarquee.setOrigin(0.5f, 0.5f);
+	mMarquee.setPosition(mSize.x() * 0.25f, mSize.y() * 0.10f);
+	mMarquee.setMaxSize(mSize.x() * (0.5f - 2*padding), mSize.y() * 0.18f);
+	mMarquee.setDefaultZIndex(35);
+	addChild(&mMarquee);
```

Comment:
Since there are existing grid themes that are not expecting a marquee to suddenly appear, the marquee should be hidden by default.  This can be done with `mMarquee.setVisible(false)`.

Reply:
The code remains the same, I prefer to set up default values, and finally the setVisible to false as suggested